django/conf/global_settings.py

```diff
@@ -502,6 +502,17 @@
 CACHE_MIDDLEWARE_SECONDS = 600
 CACHE_MIDDLEWARE_ALIAS = 'default'
 
+#########
+# TASKS #
+#########
+
+# The task backends to use.
+QUEUES = {
```

Comment:
Sounds good, I am still wondering how one would implement something like this though: Have one connection to rabbitmq and use two queues (queue beeing an queue in the context of amqp).